code/modules/client/preference_setup/occupation/occupation.dm

```diff
@@ -405,8 +405,8 @@
 			dat += "<i><b>Alternative titles:</b> [english_list(job.alt_titles)].</i>"
 		send_rsc(user, job.get_job_icon(), "job[ckey(rank)].png")
 		dat += "<img src=job[ckey(rank)].png width=96 height=96 style='float:left;'>"
-		if(job.department)
-			dat += "<b>Department:</b> [job.department]."
+		if(LAZYLEN(job.department_refs))
+			dat += "<b>Department:</b> [SSdepartments.departments[job.department_refs[1]].title]."
```

Comment:
This really needs to be broken out into a couple of var assignments I think, if only for readability.

Reply:
It looks better now it's job.primary deparment.